### ISSUE TITLE
[BUG] Omnisearch never completes indexing of some two notes 

### ISSUE THREAD

**Issue by lemmsh** [scambier/obsidian-omnisearch#119]
Hi All,

In one of my vaults I have two notes which seemingly block omnisearch from working. It is "reindexing" them and never finishes.
I don't know how to identify the rogue notes to find out what's wrong with them

![image](https://user-images.githubusercontent.com/90259/198866981-b6e6dfe1-f1e0-4d95-9d3c-fc2515b1aa4a.png)
![image](https://user-images.githubusercontent.com/90259/198867018-8ee5741f-2aa6-473e-afe0-5cccfcb541af.png)

- Omnisearch version: 1.7.8
- Obsidian version: 1.0.3
- Operating system: Fedora 36
- Number of notes in your vault (approx.): ~150
- Other plugins that may be related to the issue: no other community plugins installed

Thanks!

**Comment by scambier** [comment]
Can open the developer console (ctrl+shift+i) and report any errors that may appear?
![image](https://user-images.githubusercontent.com/3216752/198517925-916408fb-50aa-432a-be81-453d40026de4.png)

**Comment by malkomalko** [comment]
Seeing this too on latest 1.7.8

`Uncaught (in promise) TypeError: Cannot set properties of undefined (setting 'minisearch')`

Looks like it's happening here: `this.engine` is undefined.
```
static loadTmpDataIntoMain() {
            const U = this.tmpEngine.minisearch.toJSON();
            this.engine.minisearch = a.loadJS(U, RV)
}
```

**Comment by scambier** [comment]
Build 1.7.9 should fix this

**Comment by lemmsh** [comment]
confirm, fixed for me in 1.7.9. Thank you!
